app/src/main/java/org/schabi/newpipe/player/PlayerService.java

```diff
@@ -104,11 +116,10 @@ public int onStartCommand(final Intent intent, final int flags, final int startI
             return START_NOT_STICKY;
         }
 
-        if (player != null) {
-            player.handleIntent(intent);
-            player.UIs().get(MediaSessionPlayerUi.class)
-                    .ifPresent(ui -> ui.handleMediaButtonIntent(intent));
-        }
+        initializePlayer();
```

Comment:
I think it is ill-advised to move the player initialization from `onCreate` to `onStartCommand`, as this is a major change to the lifecycle.